docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,592 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        For the given range <code>(start, stop)</code>, segmentize it into 
</pre><p>"segment" instead of "segmentize"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>